xcode/Subconscious/Shared/Components/Common/AppThemeBackgroundViewModifier.swift

```diff
@@ -7,13 +7,10 @@
 
 import SwiftUI
 
-struct AppThemeBackgroundViewModifier: ViewModifier {
+struct AppThemeBackgroundView: View {
     @Environment(\.colorScheme) var colorScheme
-    
-    func body(content: Content) -> some View {
-        content
-            .background(
-                colorScheme == .dark ? DeckTheme.darkBg : DeckTheme.lightBg
-            )
+    var body: some View {
+        (colorScheme == .dark ? DeckTheme.darkBg : DeckTheme.lightBg)
```

Comment:
It doesn't seem like much, but directly rendering a `LinearGradient` as a view vs. using it as a background on a frame has a _massively_ reduced rendering cost.